amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/ActivityInformation.java

```diff
@@ -10,6 +10,9 @@
     @JsonProperty(ActivityEPConstants.AMP_ACTIVITY_ID)
     private Long ampActivityId;
 
+    @JsonProperty(ActivityEPConstants.AMP_ACTIVITY_LAAST_VERSION_ID)
+    private Long ampASctiviylastVersionId;
```

Comment:
pls fix type in `ampASctiviylastVersionId` to `ampActiviyLastVersionId`

Reply:
done